src/test/java/com/yahoo/bullet/aggregations/sketches/QuantileSketchTest.java

```diff
@@ -79,14 +82,25 @@ public static String getEnd(String range) {
         return range.substring(endStart, range.length() - 1);
     }
 
+    private QuantileSketch getDefaultQuantileSketch() {
+        return new QuantileSketch(64, 6, Distribution.Type.QUANTILE, 10, bulletRecordProvider);
+    }
+
+    private BulletRecordProvider bulletRecordProvider;
+
+    @BeforeMethod
+    private void setup() {
+        bulletRecordProvider = new BulletConfig().getBulletRecordProvider();
```

Comment:
We don't need a before method. Just use a static variable. Ditto for other test files